sass/components/_asset-card.scss

```diff
@@ -0,0 +1,107 @@
+.asset-card {
+    display: grid;
+    grid-template-areas:
+        "banner"
+        "desc"
+        "tags";
+    grid-template-rows: 12rem 5.5rem 2rem;

```

Comment:
I think we should probably rein in the height of the text section a bit (cut down on the character limit if necessary, maybe reduce the font size). The cards have _just_ a bit too much height right now. Most descriptions are short and this results in _a lot_ of wasted space. I'm down to do that in a separate PR though.

Reply:
I'll see what I can do. I picked the height based on the longer descriptions but it was mostly just playing with some values. I'm down with reducing the size a bit

Reply:
I reduced the padding a bit and limited it to 3 lines now. This is how it looks.
![image](https://user-images.githubusercontent.com/8348954/185264158-c7c35120-c9a8-4a69-b945-a030350ea1f6.png)

I tried going to 2 lines max but it looked weird.

Reply:
I dig it! One more small tweak: I do think adding back a bit more padding is "worth it".

Reply:
I increased the padding a bit and made the area for the tags slightly larger so it could have a more consistent padding.

![image](https://user-images.githubusercontent.com/8348954/185279660-f7f270b5-21ff-40c6-a336-38e190445deb.png)

Reply:
Lovely!